notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>why not combine this logic into the first method, and provide some messaging around adjustments made if estimated cores is lower than minimum</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>one is going from qps -&gt; replicas, this one is going from replicas -&gt; qps. I suppose one could estimate  replicas if qps was passed, and estimate qps if replicas was passed. I feel like that violates single responsibility, but there are a lot of replicated characters here.</p>

Reply:
ok, np. this is like fine tuning the estimate, makes sense